include/revng/EarlyFunctionAnalysis/ControlFlowGraph.h

```diff
@@ -4,6 +4,8 @@
 // This file is distributed under the MIT License. See LICENSE.md for details.
 //
 
+#pragma clang optimize off
+
```

Comment:
Aaaah! I'm a monster!

Reply:
I didn't even notice it was there until you pointed it out :)

Reply:
Still here

Reply:
It's gone now.